manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -394,11 +394,15 @@ private static function loadEnv(string $projectDir): void
             return;
         }
 
-        if (!file_exists($projectDir.'/.env')) {
-            return;
+        // Load cached env vars if the .env.local.php file exists
+        // See https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/bootstrap.php
+        if (\is_array($env = @include $projectDir.'/.env.local.php')) {
+            foreach ($env as $k => $v) {
+                $_ENV[$k] = $_ENV[$k] ?? (isset($_SERVER[$k]) && 0 !== strpos($k, 'HTTP_') ? $_SERVER[$k] : $v);
```

Comment:
This does not set the variables in `$_SERVER`, but that's where we read them from?

Reply:
This is the exact code from Symfony flex and I have verified that it works. Maybe `$_ENV` and `$_SERVER` are treated the same internally?

Reply:
The code is incomplete. I will fix it.

https://github.com/symfony/recipes/blob/master/symfony/framework-bundle/4.2/config/bootstrap.php

Reply:
Fixed in 3092d0dea687b47e4e545fd70b6696c93b1f9bc1.